examples/v1beta1/sdk/tune-train-from-func.ipynb

```diff
@@ -402,10 +441,12 @@
     }
    ],
    "source": [
-    "status = katib_client.is_experiment_succeeded(exp_name)\n",
+    "import time\n",
+    "time.sleep(120)\n",
+    "status = katib_client.is_experiment_succeeded(exp_name, namespace=namespace)\n",
```

Comment:
Can we replace fixed-time sleep with `wait_for_experiment_condition()`?

https://github.com/kubeflow/katib/blob/2b41ae62ab3905984e02123218351a703c03bf56/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L1002-L1010